src/select-menu/docs/examples/SelectMenu-multi.example

```diff
@@ -0,0 +1,46 @@
+<Manager>
```

Comment:
This is not a blocker to merge this in, but long term we hope to migrate Evergreen over to always use `@reactions/component` in examples: https://github.com/segmentio/evergreen/issues/175. Just so you are aware next time 👍

Reply:
Got it! Definitely want to get this down. Will investigate.